docs/.link-check-errors.txt

```diff
@@ -1,18 +1,5 @@
 Broken links found by `yarn docs:link-check` that exist:
 
-- https://mui.com/base-ui/react-autocomplete/hooks-api/#use-autocomplete
-- https://mui.com/base-ui/react-portal/components-api/
-- https://mui.com/material-ui/api/button/#props
-- https://mui.com/material-ui/api/input-base/#InputBase-prop-disableInjectingGlobalStyles
-- https://mui.com/material-ui/api/menu-item/#css
-- https://mui.com/material-ui/api/popper/#props
 - https://mui.com/material-ui/customization/right-to-left/
```

Comment:
I've bumped `monorepo` locally and checked that this and L5 are removed after that.